app/styles/get-started.css

```diff
@@ -0,0 +1,70 @@
+.landing {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  height: 80vh;
+  width: 70%;
+  margin: auto;
+}
+
+.landing__mainHeading {
+  font-size: 2rem;
+  color: black;
+}
+
+.landing__subHeading {
+  font-size: 1.5rem;
+  color: black;
+}
+
+.landing > * {
+  padding: .75rem;
+}
```

Comment:
Please follow BEM. I think it is not allowed.
if it is allowed, then please provide the docs.

If we are using mixed verison, then please ignore.